src/transformers/models/dac/configuration_dac.py

```diff
@@ -0,0 +1,49 @@
+"""Dac model configuration"""
+
+import numpy as np
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+class DacConfig(PretrainedConfig):
+    r"""
+    """
+
+    model_type = "dac"
+
+    def __init__(
+        self,
+        encoder_dim=64,
+        encoder_rates=[2, 4, 8, 8],
+        latent_dim=None,
```

Comment:
How is this different from `encoder_hidden_size`? If required, can we re-name it to `hidden_size` and set a suitable default?

Reply:
It is computed this way: `latent_dim = encoder_hidden_size * (2 ** len(downsampling_ratios))`. 

We can remove it from the config parameters, rename it `hidden_size`, and infer it using `encoder_hidden_size` and `downsampling_ratios`.